core/src/main/java/org/frankframework/util/TransformerPool.java

```diff
@@ -479,21 +488,18 @@ private MessageContext createMessageContext() {
 	 * When method parameter 'Result' is used, nothing will be returned.
 	 */
 	@Deprecated
-	public String deprecatedParameterTransformAction(Source s, Result r, ParameterValueList pvl) throws TransformerException, IOException {
-		return transform(s, r, pvl==null? null : pvl.getValueMap());
+	public void deprecatedParameterTransformAction(Source s, Result r, ParameterValueList pvl) throws TransformerException, IOException {
```

Comment:
Gaan we hier nu nog mee verder, hij wordt na deze aanpassing nog maar op 1 plek gebruikt. Daar zou je mogelijk ook direct `transform` aan kunnen roepen?

Reply:
Ik wist niet zeker wat jouw bedoeling was met verdere refactoring in Parameter classes maar dat kan zeker.